### PR TITLE
Different types of styles/fonts used in staticAnalysis.md #6843

### LINKED ISSUE #6843: Different types of styles/fonts used in staticAnalysis.md
Take a look the image below.

imageedit_5_5777107652

This is not the only one. It's also repeated. I think its better to use only one format, may be the 2nd one is better instead of using `#` . It's happening because of the missing space.

### PR DESCRIPTION
**Pull request by saitj99**
Fixes #6843 

**Outline of Solution**

> Tell us how you solved the issue
Added additional spaces  after #### in staticAnalysis.md as described in #6843.

### CODE REVIEW

**Comment by saitj99** [comment]
`Ready for review`

File: docs/staticAnalysis.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ The plugin for IntelliJ can be found [here](https://plugins.jetbrains.com/idea/p
 1. In the `Options` tab, set `Target JDK` to 1.7.
 1. Click `OK`.
 
-#####Suppressing PMD warnings
+##### Suppressing PMD warnings
```
Comment (AnaghHegde) [inline docs/staticAnalysis.md]: Add a new line here.

Comment (xpdavid) [inline docs/staticAnalysis.md]: @AnaghHegde Nice catch. Yes @saitj99, please add a new line for consistency.

Comment (saitj99) [inline docs/staticAnalysis.md]: @AnaghHegde @xpdavid updated in https://github.com/TEAMMATES/teammates/pull/6853/commits/74c8c4482bf5922bb6e804509fb97bdb8e9eb53e

**Pull request merged**